src/models/global.js

```diff
@@ -8,7 +8,7 @@ export default {
     sessionDescription: '',
     sessionId: '',
     collapsed: true,
-    selectedIndices: [],
+    selectedDateRange: [],
```

Comment:
Can we consider making `selectedDateRange` an object with `start` and `end` keys for readability purposes throughout the code?